docs/source/release/templates/framework.rst

```diff
@@ -0,0 +1,61 @@
+=================
+Framework Changes
+=================
+
+.. contents:: Table of Contents
+   :local:
+
+Algorithms
+----------
+
+New features
+############
+.. amalgamate:: Framework/Algorithms/New_features
+
+Bugfixes
+############
+.. amalgamate:: Framework/Algorithms/Bugfixes
+
+Fit Functions
+-------------
+
+New features
+############
+.. amalgamate:: Framework/Fit_Functions/New_features
+
+Bugfixes
+############
+.. amalgamate:: Framework/Fit_Functions/Bugfixes
+
+
+Data Objects
+------------
+
+New features
+############
+.. amalgamate:: Framework/Data_Objects/New_features
+
+Bugfixes
+############
+.. amalgamate:: Framework/Data_Objects/Bugfixes
+
+
+Python
+------
+
+New features
+############
+.. amalgamate:: Framework/Python/New_features
+
+Bugfixes
+############
+.. amalgamate:: Framework/Python/Bugfixes
+
+
+Installation
+------------
+
+MantidWorkbench
```

Comment:
MantidWorkbench is not mentioned in the Release Note Guide

Reply:
The Framework release notes have had a link over to the Workbench release notes for some time. As the heading is for a link rather than any release notes it shouldn't be in the Release Note Guide. The guide should only include the relevant file structure.